### ISSUE TITLE
Can we use googleAnalyticsR on colab?

### ISSUE THREAD

**Issue by ammpfc** [MarkEdmondson1234/googleAnalyticsR#408]
Hi,

I would like to make an R project with googleAnalyticsR on google collab to share and use on the browser.

One of the limitations that I have found regards authentication.

running ga_auth() the message to authenticate on the browser is received however no pop-up is opened to proceed with this step.
Could you please advise me on how to overcome this? Can I use another strategy to perform the authentication to pull data from GA?

Thank you so much for the attention

**Comment by MarkEdmondson1234** [comment]
Yes, I recommend you should consider it a server application and use a service key json file instead. Add the email of the service key as a user to your GA. See gargle website for details.
